processor/elasticinframetricsprocessor/processor_test.go

```diff
@@ -111,6 +111,85 @@ func TestProcessMetrics(t *testing.T) {
 	}
 }
 
+func TestDropOriginalMetrics(t *testing.T) {
+	testCases := []struct {
+		name                   string
+		cfg                    *Config
+		createMetrics          func() pmetric.Metrics
+		expectedPodname        string
+		expectedMetricName     string
+		expetedMetricValue     float64
+		expetedLengthOfMetrics int
+	}{
+		{
+			name: "ProcessMetrics when AddK8sMetrics is enabled and DropOriginal is enabled",
+			cfg:  &Config{AddK8sMetrics: true, DropOriginal: true},
+			createMetrics: func() pmetric.Metrics {
+				md := pmetric.NewMetrics()
+				rm := md.ResourceMetrics().AppendEmpty()
+				rm.Resource().Attributes().PutStr("k8s.pod.name", "test-pod")
+				sm := rm.ScopeMetrics().AppendEmpty()
+				// Creating first metric
+				metric1 := sm.Metrics().AppendEmpty()
+				metric1.SetName("k8s.pod.cpu_limit_utilization")
+				dp1 := metric1.SetEmptyGauge().DataPoints().AppendEmpty()
+				dp1.Attributes().PutBool("otel_remapped", true)
+				dp1.SetDoubleValue(0.5)
+				//Creating second metric
+				// This metric does not have the otel_remapped:true attribute set, so it should be dropped
+				metric2 := sm.Metrics().AppendEmpty()
+				metric2.SetName("k8s.test")
+				dp2 := metric2.SetEmptyGauge().DataPoints().AppendEmpty()
+				dp2.SetDoubleValue(0.5)
+				return md
+			},
+			expectedPodname:        "test-pod",
+			expectedMetricName:     "k8s.pod.cpu_limit_utilization",
```

Comment:
If we drop the initial metrics, why are we expecting the same initial naming? Shouldn't it be `kubernetes.pod.cpu.usage.limit.pct`?

Reply:
TBH I had the same question ! I will have to investigate for this and see why it does not rename/ remap it. Le you know

Reply:
@rogercoll I think with yours PR or my fix here we are ok